### PR TITLE
Add Tagger overview + run mkDocs script in travis

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

**Comment by elouanKeryell-Even** [comment]
@MohamedHamouGisaia tell me when you are done pushing commits :grin:

File: docs/arlas-tagger-overview.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# ARLAS Tagger API Overview
+
+The ARLAS Tagger offers 3 APIs:
+
+- a `write` API for [tagging](arlas-api-tagging.md), meaning adding (or removing) a value to a field in ARLAS `collections` (`http://.../arlas_tagger/write/`)
+- an `status` API for monitoring the `tagging` operation status (`http://.../arlas_tagger/status/`
```
Comment (elouanKeryell-Even) [inline docs/arlas-tagger-overview.md]: Typo:

> an `status`

should be

> a `status`

File: mkDocs.sh [hunk 2]
```diff
@@ -124,7 +124,7 @@ if [[ ! -f ${BASEDIR}/target/generated-docs/typescript-doc/classes/_api_.writeap
     echo 'File "_api_.writeapi.md" is not generated, aborting.'
     exit -1
 fi
-if [[ ! -f ${BASEDIR}/target/generated-docs/reference.md ]] ; then
-    echo 'File "reference.md" is not generated, aborting.'
+if [[ ! -f ${BASEDIR}/target/generated-docs/tagger_reference.md ]] ; then
+    echo 'File "tagger_reference.md" is not generated, aborting.'
```
Comment (elouanKeryell-Even) [inline mkDocs.sh]: IMO past tense sounds better, i.e. `File [...] was not generated`

File: docs/arlas-tagger-overview.md (hunk 1 above)
Comment (MohamedHamouGisaia) [reply docs/arlas-tagger-overview.md]: done

File: mkDocs.sh (hunk 2 above)
Comment (MohamedHamouGisaia) [reply mkDocs.sh]: Done

**Pull request merged**